lib/index.d.ts

```diff
@@ -0,0 +1,36 @@
+/* Type definitions for mobx-event-bus */
+/* Version 0.1.0 */
+/* Definitions by: Igor Savin https://github.com/kibertoad */
+
+declare interface EventBus {
+
+    /**
+     *
+     *  @function
+     */
+    metadata(): object
+
+    /**
+     * Registers subscriber to receive events
+     * @function
+     * @param {object} subscriber
+     */
+    register(subscriber: object): void;
+
+    /**
+     * Post event with given topic and payload
+     * @function
+     * @param {string} topic
+     * @param {Object} payload
+     */
+    post(topic: string, payload: object): void;
+
+}
+
+export const eventBus : EventBus;
+
+interface Selector<Event> {
+    (event: Event): boolean;
+}
+
+export function subscribe<Event>(topic: string, selector?: Selector<Event>)
```

Comment:
Would it make more sense to define an `Event` type with `topic` and `payload` properties and then request only the Event's payload type using this generic? This seems to be the API as described in [API docs](https://github.com/7upcat/mobx-event-bus#api-documents) of this project.